settings.md

```diff
@@ -17,21 +17,38 @@ title: Kint settings
 
 # Kint settings
 
-These are global Kint settings. For plugin settings please see the [plugins page]({{ site.baseurl }}/plugins/).
+These are global Kint settings. For plugin settings please see the [plugins page]({{ site.baseurl }}/plugins/). 
 
 <section id="common" markdown="1">
 
+## Usage with a framework
```

Comment:
Did you understand what this chapter is about? 

If you have a different approach to what is being suggested there, please propose your way. Or if I missed this information somewhere else, please point it to me, cause it's not on the front page of the site.

Reply:
> Usage with a framework

The problem is that every framework has a different way of doing it. Quite a few have different ways of doing it in a single framework (IE xml/yml/php in symfony if it's added as a flex component or something)

Since the config (currently) is just a bunch of static props, I think we shouldn't be opinionated on how to config it.

I especially think giving people a copy paste `autoloader` section is asking for trouble when we get issues complaining about how "Kint broke our system!" because their application namespace wasn't `App` or their framework has multiple namespaces in the root composer.json or something.